Add information about scroll (#494)
kuzzleio/sdk-javascript: doc/7/controllers/document/search/index.md

When processing a large number of documents (i.e. more than 1000), it is advised to paginate the results using [SearchResult.next](/sdk/js/7/core-classes/search-result/next) rather than increasing the size parameter.
:::

::: warning
When using a cursor with the `scroll` option, Elasticsearch has to duplicate the transaction log to keep the same result during the entire scroll session.  

> It can lead to memory leaks in a too great scroll duration is provided or if too many scroll session are open simultaneously.  

Reviewer: ```suggestion
It can lead to memory leaks if a too great scroll duration is provided or if too many scroll sessions are open simultaneously.  
```

> It can lead to memory leaks if a too great scroll duration is provided or if too many scroll sessions are open simultaneously.  

Reviewer 2: ```suggestion
It can lead to memory leaks if a scroll duration too great is provided, or if too many scroll sessions are open simultaneously.  
```

:::

::: info
<SinceBadge version="Kuzzle 2.2.0"/>

> You can restrict the scroll session maximum duration under the `services.storage.maxScrollDuration` configuration key and the maximum number of simultaneously opened scroll session under the `services.storage.maxScroll` configuration key.

Reviewer: ```suggestion
You can restrict the scroll session maximum duration under the `services.storage.maxScrollDuration` configuration key and the maximum number of simultaneously opened scroll sessions under the `services.storage.maxScroll` configuration key.
```
